### PR TITLE
Misc fixes

### PR DESCRIPTION
**Pull request by cperciva**

### CODE REVIEW

File: tests/s3/test_s3.sh [hunk 1]
```diff
@@ -5,7 +5,7 @@ S3=../../s3/s3
 TESTS3=./test_s3
 TMPDIR=`pwd`/tmp
 SOCKS3=$TMPDIR/sock_s3
-REGION=s3-us-west-2
```
Comment (gperciva) [inline tests/s3/test_s3.sh]: huh, no wonder I couldn't get `tests/kvlds-s3/` to work!

**Comment by gperciva** [comment]
LGTM

File: tests/s3/test_s3.sh (hunk 1 above)
Comment (cperciva) [reply tests/s3/test_s3.sh]: Well, also you don't own the `kivaloo-test` S3 bucket.  S3 buckets have a global namespace, unlike pretty much everything else in AWS.

Comment (gperciva) [reply tests/s3/test_s3.sh]: I figured out that part, and appended `-gperciva`.

**Pull request merged**